icu4c/source/i18n/number_formatimpl.h

```diff
@@ -79,14 +79,22 @@ class NumberFormatterImpl : public UMemory {
      * Synthesizes the output string from a MicroProps and DecimalQuantity.
      * This method formats only the main number, not affixes.
      */
-    static int32_t writeNumber(const MicroProps& micros, DecimalQuantity& quantity,
-                               FormattedStringBuilder& string, int32_t index, UErrorCode& status);
+    static int32_t writeNumber(
```

Comment:
Please check indentation in this PR -- 4 spaces each step, not 2.

Reply:
I think I fixed it. My editor was detecting 2-space indentation in some files.